infra/gcb/wait_for_build.py

```diff
@@ -59,7 +66,14 @@ def main():
   cloudbuild = build('cloudbuild', 'v1', credentials=credentials)
 
   success = wait_for_build(args.build_id, args.project)
-
+  if success:
```

Comment:
nit: the value of `success` is not depended on anywhere else. let's just do

```python
if wait_for_build(...):
```

Same with Line 76

Reply:
done